scripts/stats.php

```diff
@@ -136,51 +170,52 @@
   <video onplay='setLiveStreamVolume(0)' onended='setLiveStreamVolume(1)' onpause='setLiveStreamVolume(1)' controls poster=\"$filename.png\" title=\"$filename\"><source src=\"$filename\"></video></td>
   </tr>
     </table>
-  <p>Loading Images from <a href=\"https://commons.wikimedia.org/w/index.php?search=$linkname&title=Special:MediaSearch&go=Go&type=image\" target=\"_blank\">Wikimedia Commons</a></p>", '6096');
+  <p>Loading Images from Flickr</p>", '6096');
   
   echo "<script>document.getElementsByTagName(\"h3\")[0].scrollIntoView();</script>";
   
   ob_flush();
   flush();
-  $imagelink = "https://commons.wikimedia.org/w/index.php?search=$linkname&title=Special:MediaSearch&go=Go&type=image";
-  $homepage = file_get_contents($imagelink);
-  preg_match_all("{<img\\s*(.*?)src=('.*?'|\".*?\"|[^\\s]+)(.*?)\\s*/?>}ims", $homepage, $matches, PREG_SET_ORDER);
-  foreach ($matches as $val) {
-      $pos = strpos($val[2],"/");
-      $link = substr($val[2],1,-1);
-      if($pos !== 1 && strpos($link, "upload") == true && strpos($link, "CentralAutoLogin") == false)
-          echo "<img src=\"$link\">";
+  $flickrjson = json_decode(file_get_contents("https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=2158f6c5d66e89778bd0d340953f1bcf&text=\"".str_replace('_', '+', $comname)."\"&license=2%2C3%2C4%2C5%2C6%2C9&sort=relevance&per_page=15&format=json&nojsoncallback=1"), true)["photos"]["photo"];
```

Comment:
Hmm, was this someone's APi key leaked+hardcoded here? @mcguirepr89 + @ehpersonal38

Reply:
Yes!!! I imagine that belongs to @ehpersonal38 I would assume that key is meant to be `$config['FLIKR_API_KEY']` or something along those lines.

I should be able to clean it up